tests/test_depsolver.py

```diff
@@ -212,7 +212,11 @@ def test_run(pulp):
         in_pulp_repos=[repos[1]],
     )
 
-    with Depsolver([dep_item_1, dep_item_2], [repo_srpm]) as depsolver:
+    module_list = {
```

Comment:
Slightly weird to call it `module_list` when it's not a list.  I'd usually prefer to keep the type names out of the variable names and call it something like `modules`.

Reply:
Thanks! Changed to modules.